docs/modules/con_hollow-jar.adoc

```diff
@@ -46,7 +46,7 @@ ifndef::product[]
 full:: The main Java EE related capabilities
 endif::[]
 web:: Functionality focused on web technologies
-microprofile:: Functionality defined by all Eclipse MicroProfile specifications
+microprofile:: Functionality defined by all Eclipse Eclipse MicroProfile specifications
```

Comment:
I don't think so :-)

Reply:
Oh, snap! didn't catch that one. Thanks for spotting it:)
```suggestion
microprofile:: Functionality defined by all Eclipse MicroProfile specifications
```